test/unit/master/test_cluster_master.py

```diff
@@ -128,17 +132,32 @@ def test_handle_result_reported_from_slave_does_nothing_when_build_is_canceled(s
         build_id = 1
         slave_url = "url"
         build = Build(BuildRequest({}))
-        build.handle_subjob_payload = Mock()
```

Comment:
Fun fact: because of the way this test was written it was still passing even though `handle_subjob_payload` was no longer a real method.